config/tracking_beam_test/tracking_beam_frame_to_ndisk.yaml

```
@@ -0,0 +1,77 @@
+##########################################
+#
+# tracking_beam_frame_to_ndisk.yaml
+#
+# Configuration for recording the tracking
+# beam to n disks.
+#
+# Author: Jing Luo
+#
+##########################################
+---
+type: config
+log_level: debug2
+cpu_affinity: [0,1,2,3,4,5,6,7,8,9,10,11,12,13,14,15]
+samples_per_data_set: 49152
+num_pol: 2
+buffer_depth: 105
+sub_frames_per_merged_frame: 11
+# The size of metadata with freq ID
+freq_id_metadata_size: 72
+
+telescope:
+  name: CHIMETelescope
+  require_gps: true
+  query_gps: true
+  require_frequency_map: true
+  query_frequency_map: true
+  allow_default_frequency_map: false
+
+dataset_manager:
+  use_dataset_broker: false
+
+beam_metadata_pool:
+  kotekan_metadata_pool: FreqIDBeamMetadata
+  num_metadata_objects: 4096
+
+beam_recv_buffer:
+  kotekan_buffer: standard
+  num_frames: 2048
+  metadata_pool: beam_metadata_pool
+  frame_size: samples_per_data_set * num_pol
+
+merged_buffer:
+  kotekan_buffer: standard
+  num_frames: 10
```

I would make this at least 30, you will always have 8 frames busy with writes, and you want enough buffer to handle temporary slow downs in the disks.   Make sure you increase the pool size as well when you do this. 
```suggestion
  num_frames: 30
```